contracts/SB_scripts/rinkeby/Rinkeby_0007_interestRates.sol

```diff
@@ -93,11 +93,8 @@ contract Rinkeby_0007_interestRates {
     function disableAllLockProducts(Locker target) internal {
         uint32 productCount = uint32(target.getLockProductCount());
         for (uint32 i = 0; i < productCount; i++) {
-            uint32 perTermInterest;
-            uint32 durationInSecs;
-            uint32 minimumLockAmount;
             bool isActive;
-            (perTermInterest, durationInSecs, minimumLockAmount, isActive) = target.lockProducts(i);
+            (, , , isActive) = target.lockProducts(i);
```

Comment:
why are we modifying scripts which we already executed?

Reply:
We unfortunately had to modify one: only this one rinkeby script tried to be too smart, and read data from the contract (via automatic getters), so we had to modify this slightly to compile. The change does not impact functionality, only there is one more number we ignore. Also i removed all the unused variables, also a noop change. (We only care about the isActive flag, all other properties are ignored)

Reply:
I see. See my prev comment about potentially archiving old SB scripts.

Reply:
Are we archiving SB scripts and rolling this change back?

Reply:
Archiving SB scripts should be done on a separate branch - as it is not really related to the loan-margin feature, and merge to this feature branch via staging.  After that this file will not even exist in its current location, making this change completely irrelevant.

Reply:
done: we've archived the old scripts, rolled back above change